contracts/RenExSettlement.sol

```diff
@@ -74,28 +77,30 @@ contract RenExSettlement is Ownable {
         Orderbook _orderbookContract,
         RenExTokens _renExTokensContract,
         RenExBalances _renExBalancesContract,
-        uint256 _submissionGasPriceLimit
+        uint256 _submissionGasPriceLimit,
+        address _newSlasherAddress
     ) public {
         orderbookContract = _orderbookContract;
         renExTokensContract = _renExTokensContract;
         renExBalancesContract = _renExBalancesContract;
         submissionGasPriceLimit = _submissionGasPriceLimit;
+        slasherAddress = _newSlasherAddress;
     }
 
     /********** UPDATER FUNCTIONS *********************************************/
 
     function updateOrderbook(Orderbook _newOrderbookContract) public onlyOwner {
-        emit OrderbookUpdated(orderbookContract, _newOrderbookContract);
+        // emit OrderbookUpdated(orderbookContract, _newOrderbookContract);
```

Comment:
Why are all the events disabled?

Reply:
The compiled bytecode exceeded the size limit so events have been temporarily disabled. They will be re-enabled when the contract is split.